core-feature-pack/src/main/resources/content/standalone/deployments/README.txt

```diff
@@ -36,9 +36,9 @@ of command telling the scanner to deploy, undeploy or redeploy content.
 
 
 Auto-deploy mode and manual deploy mode can be independently configured for
-zipped deployment content and exploded deployment content. This is done
-via the "auto-deploy" attributes on the deployment-scanner element in the
-standalone.xml configuration file:
+zipped deployment content and exploded deployment content. This is done by
+editing the appropriate "auto-deploy" attributes on the deployment-scanner
+element in the standalone.xml configuration file:
 
 <deployment-scanner scan-interval="5000" relative-to="jboss.server.base.dir"
     path="deployments" auto-deploy-zipped="true" auto-deploy-exploded="false"/>
```

Comment:
hum I discussed this with Pavel and it seemed to me that the issue was more in the auto-deploy-zipped and auto-deploy-exploded attributes not being present in the standalone.xml.